### PR TITLE
feat: send emails about successfull verification

### PR DESCRIPTION
**Pull request by nadiia-tsytsylina**
## Describe your changes
 add function for sending emails to successfully verified users, depending on role
 
## Issue ticket code (and/or) and link
- [Link to JIRA ticket](https://homecareaid.atlassian.net/browse/AC-81)

Caregiver email:
image

Seeker email:
image

### **General**
- [x] Assigned myself to the PR
- [x] Assigned the appropriate labels to the PR
- [x] Assigned the appropriate reviewers to the PR
- [x] Updated the documentation
- [x] Performed a self-review of my code
- [x] Types for input and output parameters
- [x] Don't have "any" on my code
- [x] Used the try/catch pattern for error handling
- [x] Don't have magic numbers
- [x] Compare only with constants not with strings
- [x] No ternary operator inside the ternary operator
- [x] Don't have commented code
- [x] no links in the code, env links should be in env file (for example: server url), constant links (for example default avatar URL) should be in constant file.
- [x] Used camelCase for variables and functions
- [x] Date and time formats are on the constants
- [x] Functions are public only if it's used outside the class
- [x] No hardcoded values
- [ ] covered by tests
- [x] Check your commit messages meet the [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/).

### Backend
- [ ] Swagger documentation updated
- [x] Database requests are optimized and not redundant
- [ ] Unit tests written
- [x] use ConfigService instead of process.env
- [ ] use transactions if there is a call chain that mutates data in different tables
- [ ] use @index decorator for frequently requested data

### CODE REVIEW

File: src/modules/auth/auth.service.ts [hunk 1]
```diff
@@ -190,6 +208,40 @@ export class AuthService {
     }
   }
 
+  private async sendVerifiedEmail(email: string, role: string): Promise<void> {
```
Comment (Simpsonkk) [inline src/modules/auth/auth.service.ts:211]: Please, get rid of duplication in this method.

Comment (nadiia-tsytsylina) [reply src/modules/auth/auth.service.ts:211]: thank you. fixed

**Comment by Simpsonkk** [comment]
Approved.

**Review by Simpsonkk — APPROVED** [review]

**Comment by lyudad** [comment]
Screenshot 2023-11-23 at 22 19 49

**Comment by lyudad** [comment]
Screenshot 2023-11-23 at 22 20 26

**Comment by nadiia-tsytsylina** [comment]
> Screenshot 2023-11-23 at 22 19 49

Can you please clarify what exactly should be corrected in the text?
CrtrlChamps instead of Afyanex Care? Or delete "team" from the last row?

**Comment by Simpsonkk** [comment]
> > Screenshot 2023-11-23 at 22 19 49
> 
> Can you please clarify what exactly should be corrected in the text? CrtrlChamps instead of Afyanex Care? Or delete "team" from the last row?

Should be our team name

File: src/common/enums/error-message.enum.ts [hunk 2]
```diff
@@ -9,4 +9,5 @@ export enum ErrorMessage {
   UserNotExist = "An account matching that email doesn't exist",
   UserEmailAndPhoneAlreadyExists = 'User with these email and phone already exists',
   BadLoginCredentials = 'Failed to sign in due to bad credentials',
+  FailedSendSuccessEmail = 'Failed to send email',
```
Comment (fedasevich) [inline src/common/enums/error-message.enum.ts]: we already have that error in email module enum

**Comment by nadiia-tsytsylina** [comment]
fix: 
deleted unsubscribe, 
updated text,
deleted unnecessary error message,
updated .env

image

image

File: src/common/enums/error-message.enum.ts (hunk 2 above)
Comment (nadiia-tsytsylina) [reply src/common/enums/error-message.enum.ts]: fixed

**Review by lyudad — APPROVED** [review]

**Comment by lyudad** [comment]
кстати текста так и не поменялись(

**Comment by nadiia-tsytsylina** [comment]
> кстати текста так и не поменялись(

Я не поняла, что именно нужно поменять. Посоветовалась с Любой и убрала слово "team" в конце, где было Afyanex Care Team. Нужно было поменять Afyanex на CtrlChamps?

**Pull request merged**